backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -179,12 +179,36 @@ export class ContributionResolver {
   async listAllContributions(
     @Args()
     { currentPage = 1, pageSize = 5, order = Order.DESC }: Paginated,
+    @Arg('statusFilter', () => [ContributionStatus], { nullable: true })
+    statusFilters: ContributionStatus[] | null,
   ): Promise<ContributionListResult> {
+    const where: {
+      contributionStatus?: FindOperator<string> | null
+    } = {}
+    const typeStatus = Object.values(ContributionStatus)
+    if (statusFilters !== null) {
```

Comment:
can be undefined. Therefore:
```suggestion
    if (statusFilters && statusFilter.length) {
```